src/pages/authentication/PickNamePage.tsx

```diff
@@ -80,9 +86,9 @@ const PickYourNamePage: React.FC = () => {
     fetchVolunteers();
   }, [user]);
 
-  const handleNameChange = (
+  const handleNameChange =  (
     _event: React.SyntheticEvent,
-    value: Volunteer | null,
+    value: VolunteerIdName | null,
```

Comment:
Can we leave it as Volunteer? The name is cleaner without the encapsulated fields.